core/server/api/utils.js

```diff
@@ -245,17 +246,18 @@ utils = {
         };
     },
 
-    trimAndLowerCase: function trimAndLowerCase(params) {
+    trimAndLowerCase: (params) => {
         params = params || '';
         if (_.isString(params)) {
             params = params.split(',');
         }
 
-        return _.map(params, function (item) {
+        return params.map((item) => {
             return item.trim().toLowerCase();
         });
     },
 
+    // Not converted to ES6 arrow functions as functions use `this` context to access object function(trimAndLowerCase)
```

Comment:
@kirrg001 As a general rule, we are converting all same-name object functions to arrow functions. This will break for object functions which try to access other functions in object with this, which below functions do to access trimAndLowerCase, so they have not been updated to arrow functions. Is it ok to keep them as exceptions to the rule as special case?

Reply:
Hmm so what happens is the context of the fn is bind to the file context (the outer definition from `utils`). Arrow functions do that by default.

If we would use minimal method definitions, it should work. The fn context will be the utils object.

```
obj = {
  fn() {}
}
```

But that means, we probably should use this style anywhere we use objects with functions.

Reply:
Yeah that should work universally, I'll update the api files to this.

Reply:
👍